netdata-installer.sh

```diff
@@ -1369,6 +1387,8 @@ fi
 # 1 - version1 > version2
 # 2 - version2 > version1
 # 3 - error
+
+# shellcheck disable=SC2086
```

Comment:
Unlike the others, this one is probably fine since there are multiple lines in the function that need it.